elasticsearch/src/main/java/org/hibernate/search/elasticsearch/logging/impl/Log.java

```diff
@@ -189,4 +190,45 @@
 	)
 	void unsupportedDynamicBoost(Class<?> boostStrategyType, Class<?> entityType, String fieldPath);
 
+	@Message(id = ES_BACKEND_MESSAGES_START_ID + 33,
+			value = "Validation failed for schema of index '%1$s'"
+	)
+	SearchException schemaValidationFailed(String indexName, @Cause Exception cause);
```

Comment:
Are you sure this is going to explain what's wrong with the schema? I'd rather not rely on nested (cause) as people hate deep stack traces.